libcudacxx/docs/extended_api.md

```diff
@@ -23,8 +23,6 @@ nav_order: 3
 
 {% include_relative extended_api/memory_resource.md %}
 
-{% include_relative extended_api/ptx.md %}
```

Comment:
Shouldnt it be added somewhere or was this always superfluous

Reply:
This was superfluous. The PTX instructions page was already on a separate page. This command included it (again) at the bottom of the extended API page: https://nvidia.github.io/cccl/libcudacxx/extended_api.html#ptx-instructions